### ISSUE TITLE
Conda does not install `ray` by default

### ISSUE THREAD

**Issue by suzhoum** [autogluon/autogluon#3920]
`mamba install -c conda-forge autogluon` does not install `ray` due to the fact that we are pinning its dependency `grpcio` to a lower version in conda https://github.com/conda-forge/autogluon.core-feedstock/blob/f08041aeeeff124bc3dd0742d91a0af691e468c5/recipe/meta.yaml#L44-L46
We should add that back once the [upstream fix](https://github.com/ray-project/ray/pull/42698) in ray is released, before we add support to newer versions of ray in https://github.com/autogluon/autogluon/pull/3774

**Comment by suzhoum** [comment]
Additionaly, `ray` osx file only started to exist starting from [2.7.1](https://anaconda.org/conda-forge/ray-default/files?version=2.7.1). Since AutoGluon moved to noarch on conda, we should make sure we pin the version to `>=2.7.1`

**Comment by AnirudhDagar** [comment]
@suzhoum do we need to make any changes in the conda release, or this is now automatically fixed given we are on a ray version >=2.7.1?

**Comment by suzhoum** [comment]
Yes I'm working on conda release right now. We need to update the conda recipe.
